manifest.json

```diff
@@ -51,6 +51,9 @@
     }, {
         "matches": ["*://music.yandex.ru/*", "*://radio.yandex.ru/*"],
         "js": [ "connectors/yandex-music.js" ]
+    }, {
```

Comment:
Sorry, I've changed the extension layout a bit while this PR was pending. Now this info should be in `background/connectors.js`.